res/css/views/elements/_Field.scss

```diff
@@ -21,25 +21,29 @@ limitations under the License.
     margin: 1em 0;
 }
 
-.mx_Field input {
+.mx_Field input,
+.mx_Field select {
     font-weight: normal;
     border-radius: 4px;
     transition: border-color 0.25s;
     border: 1px solid $input-border-color;
     padding: 8px 9px;
 }
 
-.mx_Field input:focus {
+.mx_Field input:focus,
+.mx_Field select:focus {
     outline: 0;
     border-color: $input-focused-border-color;
 }
 
-.mx_Field input::placeholder {
+.mx_Field input::placeholder,
+.mx_Field select::placeholder {
```

Comment:
Does `<select>` support placeholders? If not, then these placeholder related styles shouldn't be needed.

Reply:
looks like it doesn't, which is more annoying than anything :(

Will remove the redundant styles.